images/go/image.yaml

```diff
@@ -13,11 +13,11 @@ versions:
       config: configs/1.19-dev.apko.yaml
       extractTagsFrom:
         package: go-1.19
-        prefix: dev-
         exclude:
           - 1
   - apko:
       config: configs/latest-dev.apko.yaml
       extractTagsFrom:
-        prefix: dev-
```

Comment:
without a prefix, this will overwrite the latest one. we can just remove `extractTagsFrom:` altogether

Reply:
fixed